src/components/CompareResults/EditTitleInput.tsx

```diff
@@ -0,0 +1,112 @@
+import { useEffect, useState, useCallback } from 'react';
+
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import FormControl from '@mui/material/FormControl';
+import TextField from '@mui/material/TextField';
+import { Form } from 'react-router-dom';
+
+interface EditTitleInputProps {
+  compact: boolean;
+  value: string;
+  handleToggle: () => unknown;
```

Comment:
It might be more readable and easier to reason about to call this `onCancel` (which it is :-) ).